VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -2,36 +2,33 @@
 
 import android.app.Activity;
 import android.os.Bundle;
-import android.util.DisplayMetrics;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import com.android.volley.toolbox.NetworkImageView;
+import com.bumptech.glide.Glide;
 import com.google.inject.Inject;
-import org.edx.mobile.discussion.CourseDiscussionInfo;
 
 import org.edx.mobile.R;
 import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.discussion.CourseDiscussionInfo;
 import org.edx.mobile.logger.Logger;
 import org.edx.mobile.model.api.CourseEntry;
 import org.edx.mobile.model.api.EnrolledCoursesResponse;
 import org.edx.mobile.task.GetCourseDiscussionInfoTask;
 import org.edx.mobile.third_party.iconify.IconView;
 import org.edx.mobile.third_party.iconify.Iconify;
+import org.edx.mobile.util.images.TopAnchorFillWidthTransformation;
 import org.edx.mobile.view.common.TaskProcessCallback;
 
 import roboguice.fragment.RoboFragment;
+import roboguice.inject.InjectView;
 
 public class CourseDashboardFragment extends RoboFragment {
-
-    private TextView courseTextName;
-    private TextView courseTextDetails;
-    private GetCourseDiscussionInfoTask getCourseDiscussionInfoTask;
-
-    public static interface ShowCourseOutlineCallback{
+    public interface ShowCourseOutlineCallback{
```

Comment:
Interfaces are always static, so there's no point in explicitly declaring them as such :wink: